tests/roxctl/bats-tests/cluster/crs-lifecycle.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+setup_file() {
+    local -r roxctl_version="$(roxctl-development version || true)"
+    echo "Testing roxctl version: '${roxctl_version}'" >&3
+
+    command -v curl || skip "Command 'curl' required."
+    [[ -n "${API_ENDPOINT}" ]] || fail "Environment variable 'API_ENDPOINT' required"
+    [[ -n "${ROX_PASSWORD}" ]] || fail "Environment variable 'ROX_PASSWORD' required"
+
+    export crs_name="crs-${RANDOM}-${RANDOM}"
+    out_dir="$(mktemp -d)"; export out_dir
+    export ROX_CLUSTER_REGISTRATION_SECRETS=true
+}
+
+teardown() {
+    if [ -n "${out_dir:-}" ]; then
+        rm -rf "${out_dir}"
+    fi
+}
+
+@test "CRS can be issued" {
```

Comment:
I have added two tests for failure situations. The second point is not easy to test right now (in a meaningful way, I mean). But this will change once I have changed the backend CRS issuing implementation so that, e.g. name duplicates are disallowed.